dashboard/src/components/hackathon/HackathonTeamMember.vue

```
@@ -67,7 +67,7 @@
           class="flex items-center w-full justify-between gap-2"
         >
           <div class="flex gap-2 items-center">
-            <img :src="member.profile_photo" class="w-8 h-8 rounded-full" />
+            <img :src="member.profile_photo" :alt="`Profile photo of ${member.full_name}`" class="w-8 h-8 rounded-full" />
```

ESLint

[error] Replace `·:src="member.profile_photo"·:alt="`Profile·photo·of·${member.full_name}`"·class="w-8·h-8·rounded-full"` with `⏎··············:src="member.profile_photo"⏎··············:alt="`Profile·photo·of·${member.full_name}`"⏎··············class="w-8·h-8·rounded-full"⏎···········`

(prettier/prettier)